content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: blog
+title: "How the sausage is made: the Kubernetes 1.11 release interview, from the Kubernetes Podcast"
+date:  Monday, July 16, 2018
+---
+
+<b>Author</b>: Craig Box (Google)
+
+At KubeCon EU, my colleague Adam Glick and I were pleased to announce the [Kubernetes Podcast from Google](https://kubernetespodcast.com/). In this weekly conversation, we focus on all the great things that are happening in the world of Kubernetes and Cloud Native. From the news of the week, to interviews with people in the community, we help you stay up to date on everything Kubernetes.
+
+We [recently had the pleasure of speaking](https://kubernetespodcast.com/episode/010-kubernetes-1.11/) to the release manager for Kubernetes 1.11, Josh Berkus from Red Hat, and his deputy, the release manager for the upcoming 1.12, Tim Pepper from VMware.  
```

Comment:
do we really need to add `his deputy` here? :)
>,and the release manager for the upcoming 1.12,

there is definitely a release lead for a release, but i doubt they call anyone else on the release team a deputy.
the is the notion of a "shadow", but Tim wasn't shadowing Josh for 1.11.

Reply:
Formally Lubomir is correct as I did not shadow Josh's lead.